screens/auth/home/tabs/settings/components/DeleteAccountBottomSheet.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {
+  type Ref,
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import {Text, useColorScheme, View, StyleSheet} from 'react-native';
+import BottomSheetModal, {
+  BottomSheetTextInput,
+  BottomSheetView,
+} from '@gorhom/bottom-sheet';
+import type {BottomSheetMethods} from '@gorhom/bottom-sheet/lib/typescript/types';
+import auth from '@react-native-firebase/auth';
+import {Layouts} from '@styles/layouts';
+import {getAppColorStyles} from '@styles/colors';
+import {useIsFocused} from '@react-navigation/native';
+
+import {FirebaseErrorMap} from '../../../../../../utils/firebase/ErrorTranslation';
+import {UnAuthStyles} from '../../../../../unauth/UnAuthStyles';
+
+import {DeleteBottomSheetButton} from './DeleteBottomSheetButton';
+
+interface DeleteAccountBottomSheetProps {
+  bottomSheetController: React.RefObject<BottomSheetMethods> | undefined;
+}
+export const DeleteAccountBottomSheet = ({
+  bottomSheetController,
+}: DeleteAccountBottomSheetProps) => {
+  const colorScheme = useColorScheme();
+  const {surfaceStyle, textStyle, bottomSheetBackgroundStyle} =
+    getAppColorStyles(colorScheme);
+
+  const [password, setPassword] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+  const user = useMemo(() => {
+    return auth().currentUser;
+  }, []);
+
+  useEffect(() => {
+    setErrorMessage('');
+  }, []);
```

Comment:
This can be deleted - it only sets the `errorMessage` on initial render to be an empty string, but we pass an empty string as initial value anyway